docs/conformance.md

```diff
@@ -200,100 +207,24 @@ of the SPDX.
 This compliance point facilitates interchange of minimal licencing information
 when produced by tools supporting SPDX.
 
-## 5.11 Extension Profile compliance point <a name="5.11"></a>
```

Comment:
When was it determined to remove this section?

Reply:
Ooops, that was not the intent.

Reply:
In this section, we still have to figure out the standard naming convention of the SPDX files (Lines 286-295) -- if multiple extensions (`.spdx.json`) are ok for OMG. 

See issue: https://github.com/spdx/spdx-spec/issues/987 We can do it in a seperate PR.